docs/content/how-to/disconnected/disconnected-workarounds.md

```diff
@@ -0,0 +1,78 @@
+# Disconnected workarounds
+
+## Make ImageContentSourcePolicies work using image tags
+The ImageContentSourcePolicies (ICSP) have not exposed the API to handle the parameter `mirror-by-digest-only`, so in order to do that, we need to manually create a MachineConfig change to be applied in all the HostedCluster workers. This change will perform a similar action in the workers like an ICSP.
```

Comment:
Is this something we need to do manually?

Reply:
And if so, where do we create the MachineConfig ? the mgmt cluster or the guest cluster?

Reply:
> Is this something we need to do manually?

Yes, at least with the non-released MCE versions. If that happens once released it should be considered a bug.

> And if so, where do we create the MachineConfig ? the mgmt cluster or the guest cluster?

On the Management Cluster. This is mostly to install MCE disconnected because there are some images not uploaded with the digest.

Reply:
I consider important to people to know that this is not the good way but  something that really works in a real life situation, this is why I preffer to put it in a workaround section.